examples/sampling/graphbolt/node_classification.py

```diff
@@ -364,6 +373,21 @@ def parse_args():
         help="Dataset storage placement and Train device: 'cpu' for CPU and RAM,"
         " 'pinned' for pinned memory in RAM, 'cuda' for GPU and GPU memory.",
     )
+    parser.add_argument(
+        "--sample-mode",
+        default="sample-neighbors",
+        choices=["sample-neighbors", "sample-layer-neighbors"],
+        help="The sampling function when doing layerwise sampling.",
+    )
+    parser.add_argument(
+        "--overlap-graph-fetch",
+        type=bool,
+        default=False,
```

Comment:
```suggestion
        action='store_true',
```

Reply:
So that we can write `--overlap-graph-fetch` to enable it instead of having to write `--overlap-graph-fetch=True`.

Reply:
OK, I have changed here.